docs/troubleshooting/Comment_Bug_Reports.en.md

```diff
@@ -0,0 +1,24 @@
+# Comment and Bug Reports Form
+
+This feature allows you to directly send feedback, requests, and bug reports about the VketCloud SDK from the Unity editor.
+It provides a way to easily and anonymously share any inconveniences you experience while using the VketCloud SDK, as well as any bugs you discover during development.
+We encourage you to share your thoughts with us.
+
+If you would like a response or support from the administration, please contact us on。
```

Comment:
If you would like a direct response or support from staff, please contact us on [VketCloud/MyVket Discord](https://discord.com/invite/vsFDNTKdNZ){target=_blank}